app_server/app/controllers/StreamController.js

```diff
@@ -192,6 +200,23 @@ Class.deleteStream = function(request, reply) {
   });
 };
 
+Class.getStreamsStats = function(request, reply) {
+  logger.info(`Getting statistics for ${request.query.state} streams`);
+
+  let state = request.query.state; 
+  if (state !== 'live') {
+    return reply({});
+  }
+
+  Service.getLiveStreamsStats()
+  .then((liveStreamsStats) => {
+    if (liveStreamsStats instanceof Error) {
+      return reply(Boom.badGateway(liveStreamsStats.message));
+    }
+    return reply(liveStreamsStats);
```

Comment:
We can probably shorten this, too, using the ternary operator.

Are `all` and `done` TODOs or never-to-be-implemented?
If it's the former, add a comment here; if it's the latter, change the validator accordingly.